game/resource/English/ability/items/tooltip_bloodstone.txt

```diff
@@ -3,7 +3,7 @@
 //=============================================================================
 "DOTA_Tooltip_Ability_item_bloodstone_1"                                    "Bloodstone"
 "DOTA_Tooltip_Ability_item_recipe_bloodstone_1"                             "Bloodstone Recipe"
-"DOTA_Tooltip_Ability_item_bloodstone_1_Description"                        "<h1>Active: Pocket Deny</h1>Instantly causes you to die.<br><br>Must ground target to cast.  Cannot self cast.\n<h1>Passive: Bloodpact</h1>Begins with %tooltip_charges% charges, and gains a charge each time an enemy hero dies within %charge_range% range. <br><br>Each charge grants %tooltip_mana_per_charge% mana regeneration per second, and reduces respawn time by %respawn_time_reduction% seconds. <br><br>If the bearer dies, %heal_on_death_base% + %heal_on_death_per_charge% health per charge is restored to all nearby allies, while the Bloodstone loses a third of its charges. <br><br>Heal Radius: %heal_on_death_range%"
+"DOTA_Tooltip_Ability_item_bloodstone_1_Description"                        "<h1>Active: Pocket Deny</h1>Instantly causes you to die.<br><br>Must ground target to cast.  Cannot self cast.\n<h1>Passive: Bloodpact</h1>Begins with %tooltip_charges% charges, and gains a charge each time an enemy hero dies within %charge_range% range. <br><br>Each charge grants %tooltip_mana_per_charge% mana regeneration per second, and reduces respawn time by %respawn_time_reduction% seconds."
```

Comment:
Should be either
`<h1>Active: Pocket Deny</h1>Instantly causes you to die.<br><br>Must ground target to cast.  Cannot self cast.\n<h1>Passive: Bloodpact</h1>Begins with %tooltip_charges% charges, and gains a charge each time an enemy hero dies within %charge_range% range. <br><br>Each charge grants %mana_per_charge% mana regeneration per second, and reduces respawn time by %respawn_time_reduction% seconds. <br><br>If the bearer dies, the Bloodstone loses a third of its charges.`
or if you wanted to edit the base dota one for accuracy it'd be
`<h1>Active: Pocket Deny</h1>Instantly causes you to die.<br><br>Must ground target to cast.  Cannot self cast.\n<h1>Passive: Bloodpact</h1>Begins with %tooltip_charges% charges, and gains a charge each time an enemy hero dies within %charge_range% range. <br><br>Each charge grants %mana_per_charge% mana regeneration per second. <br><br>If the bearer dies, the Bloodstone loses a third of its charges.`